Makefile

```diff
@@ -23,7 +23,7 @@ COMPILER ?= ido
 #   gc-eu-mq       GameCube Europe/PAL Master Quest
 #   gc-eu-mq-dbg   GameCube Europe/PAL Master Quest Debug (default)
 # The following versions are work-in-progress and not yet matching:
-#   (none currently)
+#   ntsc-1.2       N64 NTSC 1.2 (Japan/US)
```

Comment:
```suggestion
#   ntsc-1.2       N64 NTSC 1.2 (Japan/US depending on REGION)
```
and I would change `$(BASEROM_DIR)/checksum.md5` for this version to `$(BASEROM_DIR)/checksum-jp.md5` and `$(BASEROM_DIR)/checksum-us.md5`, and use either depending on REGION

This would fix the me-problem of "it's weird to not specify the region yet the rom will have one and the checksum does match one of the two"

I implemented this and also allowing either as baserom in:

https://github.com/cadmic/oot/compare/4ab79a8212f51ff961b83e427a558d28b766d27f...Dragorn421:oot:baaabd9e9fcf8c03f130226a2b70994cc080f11b

This could happen in a separate PR.

Reply:
Alright, changed to say only "Japan" for now.